omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -92,16 +83,23 @@ the OMERO installation using the Homebrew Python.
 
 To install the Python provided by Homebrew::
 
-	$ brew install python
+    $ brew install python
 
 Check that Python is working and is version 2.7::
 
     $ /usr/local/bin/python --version
     Python 2.7.9
 
-To create isolated Python environemnt you can set up and use `virtual
-environments <http://www.virtualenv.org/en/latest/>`_ to install the OMERO
-Python dependencies (see :ref:`python_dependencies`).
+The installation of OMERO via Homebrew depends upon two alternate
+repositories containing extra formulae:
+https://github.com/Homebrew/homebrew-science for the HDF5 formula and
+https://github.com/ome/homebrew-alt for all the OME-provided formulae
+and older versions of Ice. To add these, run::
+
+    $ brew tap homebrew/science
+    $ brew tap ome/alt
+
+:download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>`
```

Comment:
Not sure - this comes a bit as a surprise - why to have the script ``step_01...`` here ? It seems to do what you have just done walking through the workflow up till now, and there is no explanation about it - maybe just add ``the script below will do these and these steps for you``